staticd/static_vty.c

```diff
@@ -346,38 +452,46 @@ static int static_route(struct vty *vty, afi_t afi, safi_t safi,
 			const char *gate_str, const char *ifname,
 			const char *flag_str, const char *tag_str,
 			const char *distance_str, const char *vrf_name,
-			const char *label_str, const char *table_str)
+			const char *label_str, const char *table_str, bool bfd,
+			bool bfd_mhop, const char *bfd_profile,
+			const char *route_group, const char *local_address)
 {
 	if (!vrf_name)
 		vrf_name = VRF_DEFAULT_NAME;
 
 	return static_route_leak(vty, vrf_name, vrf_name, afi, safi, negate,
 				 dest_str, mask_str, src_str, gate_str, ifname,
 				 flag_str, tag_str, distance_str, label_str,
-				 table_str, false, NULL);
+				 table_str, false, NULL, bfd, bfd_mhop,
+				 bfd_profile, route_group, local_address);
 }
 
 /* Static unicast routes for multicast RPF lookup. */
-DEFPY_YANG (ip_mroute_dist,
-       ip_mroute_dist_cmd,
-       "[no] ip mroute A.B.C.D/M$prefix <A.B.C.D$gate|INTERFACE$ifname> [(1-255)$distance]",
-       NO_STR
-       IP_STR
-       "Configure static unicast route into MRIB for multicast RPF lookup\n"
-       "IP destination prefix (e.g. 10.0.0.0/8)\n"
-       "Nexthop address\n"
-       "Nexthop interface name\n"
-       "Distance\n")
+DEFPY_YANG(
+	ip_mroute_dist, ip_mroute_dist_cmd,
+	"[no] ip mroute A.B.C.D/M$prefix <A.B.C.D$gate|INTERFACE$ifname> [{"
```

Comment:
just checking on the proper form for the command -- should `[no]` be here throughout?